app/src/playStore/java/org/fossasia/openevent/general/search/SearchLocationActivity.kt

```diff
@@ -0,0 +1,149 @@
+package org.fossasia.openevent.general.search
+
+import android.Manifest
+import android.annotation.SuppressLint
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.location.Address
+import android.location.Geocoder
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.support.v4.app.ActivityCompat
+import android.support.v4.content.ContextCompat
+import org.fossasia.openevent.general.R
+
+import android.support.v7.widget.SearchView
+import android.view.MenuItem
+import android.view.View
+import android.widget.Toast
+import com.google.android.gms.location.LocationCallback
+import com.google.android.gms.location.LocationRequest
+import com.google.android.gms.location.LocationResult
+import com.google.android.gms.location.LocationServices
+import kotlinx.android.synthetic.main.activity_search_location.*
+import kotlinx.android.synthetic.main.fragment_login.*
+import org.fossasia.openevent.general.MainActivity
+import org.koin.android.architecture.ext.viewModel
+import timber.log.Timber
+import java.io.IOException
+import java.util.*
+
+private const val FROM_SEARCH: String = "FromSearchFragment"
+private const val TO_SEARCH: String = "ToSearchFragment"
+
+class SearchLocationActivity : AppCompatActivity() {
```

Comment:
Replicating an entire activity just for removing visibility of an element? Not a good idea